src/Components/Letter.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function selectLetter (props) {
+    props.selectLetter(props.letter);
+}
+
+const Letter = (props) => {
+    return <span onClick={() => selectLetter(props)} className={props.className? props.className : ""}>{props.letter}</span>
```

Comment:
one time you use the `selectLetter` function and one time you just use `props.letter` - best if you just `const letter = selectLetter(props)` outside of the `return` and use that